docs/server_guide.md

```diff
@@ -139,30 +140,32 @@ Finally, to improve the speed of writing to HDFS for a single partition, the val
 ### Netty
 In version 0.8.0, we introduced Netty. Enabling Netty on ShuffleServer can significantly reduce GC time in high-throughput scenarios. We can enable Netty through the parameters `rss.server.netty.port` and `rss.rpc.server.type`. Note: After setting the parameter `rss.rpc.server.type` to `GRPC_NETTY`, ShuffleServer will be tagged with `GRPC_NETTY`, that is, the node can only be assigned to clients with `spark.rss.client.type=GRPC_NETTY`.
 
-When enabling Netty, we should also consider memory related configuration, the following is an example.
+When enabling Netty, we should also consider memory related configurations, the following is an example.
 
 #### rss-env.sh
 ```
-XMX_SIZE=80g
-MAX_DIRECT_MEMORY_SIZE=60g
+XMX_SIZE=20g
+MAX_DIRECT_MEMORY_SIZE=120g
```

Comment:
You seem to modify this value.

Reply:
This is just a doc about configs when using Netty. And it is more reasonable because we don't use that much heap memory anymore when enabling Netty. I've tested this (20g) already. It works like a charm. Even maybe 10g of heap memory is enough too(I can test this later).

Reply:
OK, I got it.